### ISSUE TITLE
Pretrained models for YouTube-VOS dataset

### ISSUE THREAD

**Issue by XiaoqiangZhou** [hzxie/RMNet#15]
Thanks for sharing such great work about video object segmentation and providing the detailed instructions for the code. 

1. Unfortunately, I find that the link for pretrained YouTube-VOS model is invalid. Have you finished retraining it? I'm looking forward to this model.
2. A question about the training detail. For the training on the YouTube-VOS dataset, do you use the annotated frames or all the frames in the training set? As we know, the training set of this dataset is annotated every 5 frames, e.g. for the image id from 0 to 9, the No.0 and No.5 images are annotated with segmentation masks.

Thanks!

**Comment by hzxie** [comment]
@XiaoqiangZhou 
Sorry, I am too busy to finish it.
I just use segmented frames (i.e. every 5 frames) for the YouTube-VOS dataset.
